### ISSUE TITLE
Missing dataset ID in output files when runEA on a single SE

### ISSUE THREAD

**Issue by lgeistlinger** [waldronlab/GSEABenchmarkeR#12]
When running `runEA` on just one `SummarizedExperiment`, the results are saved without the dataId of the dataset (i.e. just as `.rds` and `.txt`)

**Comment by lgeistlinger** [comment]
For this to work you need to either explicitly set:
```
metadata(se)$dataId <- "myDatasetId"
runEA(se, ...)
```

(or, alternatively, wrap the SE in a list via 
```
exp.list <- list(myDatasetId = se)
runEA(exp.list, ...)
```
)

**Comment by lgeistlinger** [comment]
My previous comment applies for the case of a single SE and a _single_ method. I just observed when using _multiple_ methods on a single SE (which I treat internally slightly different), the dataset ID is indeed missing.

I introduced a fix for that in `GSEABenchmarkeR` v1.7.3, which you can download directly from github via:

```
BiocManager::install("waldronlab/GSEABenchmarkeR")
```
(requires that you have the `remotes` package installed)
